### DISCUSSION TITLE
Remove navigation and toc -- can't make it work?

### DISCUSSION THREAD

**Discussion by fralau** [squidfunk/mkdocs-material#2188] [Q&A] [answered]
According to [instructions](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#hide-the-sidebars) to remove navigation and the table of contents, I put this at the top of the `index.md` page:

```yaml
---
hide:
    - navigation
    - toc
---
```

I also added to the markdown extensions:

```yaml
markdown_extensions:
  - meta # title can be overriden on a per document basis
 ```

Yet it seems to have no effect on the page being served. Am I missing something ?

Or could something else interfere?

If that can help, on `pip list`:

```
mkdocs                      1.1.2
mkdocs-awesome-pages-plugin 2.4.0
mkdocs-material             6.1.4
mkdocs-material-extensions  1.0.1
mkdocs-mermaid2-plugin      0.5.1
```

**Comment by squidfunk** [comment, marked as answer]
__You need to upgrade to Material for MkDocs 6.2__

This feature was first released as part of Insiders and merged when the last funding goal was hit.

**Reply by fralau** [reply]
Yes, upgrading the package made it work automagically !

I will start playing around with the flexboxes. 😄

**Comment by fralau** [comment]
Thanks a lot for this ! 🙏 

🤦 I forgot to do the natural thing, which was to upgrade the material package. 

🤷‍♂️ But at least, I have the version requirement now, so that's in my setup file now.  

More generally, it's always the question of how to keep good track of evolutions, for users. :thinking:. Release notes help, of course. For my part, I am experimenting a solution that takes the cue from Python's official doc: for significant changes, I indicate the version "as of" directly in the documentation. That will give a cue to readers, to check if they need to upgrade.

**Reply by squidfunk** [reply]
The general advice for Material for MkDocs is to always use the latest version. I'm going to great lengths to keep stuff downward compatible and breaking changes to an acceptable minimum, but bugfixes just keep rolling in.

Also, [deprecations](https://squidfunk.github.io/mkdocs-material/deprecations/) are documented here.
